docs/source/reference/structs.rst

```diff
@@ -15,3 +15,6 @@ Structs
 
 .. autoclass:: StudySummary
     :members:
+
+.. autoclass:: TrialPruned
```

Comment:
`optuna.structs.TrialPruned` has been deprecated since 0.19.0 (c.f., https://github.com/optuna/optuna/blob/master/optuna/structs.py#L345). Please remove it from the documentation.